docs/source/02_get_started/04_configuration.md

```diff
@@ -499,6 +500,24 @@ storage:
   shared_filesystem: 10Gi
 ```
 
+## Load Balancer
+
+The kubernetes load balancer is used for exposing Qhub. Often times
+the load balancer may need to be configured with a static IP or
+additional annotations to use an internal vpc. QHub optionally
+supports these features.
+
+```yaml
+load_balancer_ip: "10.11.12.13"
+load_balancer_annotations:
+  "networking.gke.io/load-balancer-type": "Internal"
```

Comment:
Not that it really matters much, but might be a bit more tidy to organize these under a common `load_balancer` key.

Reply:
Agreed. There might be an increased number of this sort of tweak now that the terraform modules are integrated. A quick brain storm on how to organize these as they are added  would be valuable in the long run.
@costrouc I'm happy to deploy off any branch/fork so there's no rush on our deployment's account.

Reply:
Yeah I agree too. For me it was controversial to add this in that I'm hesitant to add more keys to the qhub-config.yaml. I would prefer to have it all under a single key. Will make that change